remotespark/livyclientlib/livysession.py

```diff
@@ -75,7 +75,10 @@ def create_sql_context(self):
 
         self.logger.debug("Starting '{}' sql and hive session.".format(self.kind))
 
+        self.ipython_display.writeln('Creating SqlContext as \'sqlContext\'')
```

Comment:
Nit: you could just do double quotes so that you don't have to escape single quotes:
`self.ipython_display.writeln("Creating SqlContext as 'sqlContext'")`